contributing/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Contributing"
+---
+
+## Repository Management
+
+* All new repositories should be created in the [Greenland-Net](https://github.com/QGreenland-Net) github organization using the [template](https://github.com/QGreenland-Net/template).
```

Comment:
@trey-stafford and I created a very simple template to ensure we have a consistent code of conduct and license between all repositories in this project.